plugin/src/main/java/net/codingarea/challenges/plugin/utils/bukkit/misc/Version/VersionInfo.java

```diff
@@ -1,4 +1,4 @@
-package net.codingarea.challenges.plugin.utils.bukkit.misc.version;
+package net.codingarea.challenges.plugin.utils.bukkit.misc.Version;
```

Comment:
why??

Reply:
Git seems to ignore the case of the folder path and it wasn't changed on the remote...